frontend/website/src/routes/+page.svelte

```diff
@@ -8,14 +8,97 @@
 	});
 </script>
 
-<body class="h-screen bg-gradient-to-b from-black to-neutral-800 text-white bg-no-repeat">
-	<div class="flex flex-col items-center">
-		<div class="mt-64 flex flex-col justify-center">
-			<div class="flex flex-col items-center">
-				<h1 class="font-primary font-bold text-7xl text-transparent bg-clip-text bg-gradient-to-b from-[#FF7357] to-[#FF4C6D]">Scuffle.tv</h1>
-				<p class="text-neutral-400">We’re building a streaming platform.</p>
-				<p class="text-neutral-400">A <span class="text-white">community driven streaming platform</span>, that is.</p>
+<body>
+	<div class="body container">
+		<div class="top-margin">
+			<div class="container header">
+				<h1 class="wordmark">Scuffle.tv</h1>
+				<p class="heading-description">
+					We’re building a streaming platform. 
+					<br/>
+					A <span class="highlight">community driven streaming platform</span>, that is.
+				</p>
+			</div>
+			<div class="section">
+				<h2>For Fun&trade;</h2>
+				<p>
+					Have you ever thought of making your own streaming platform? No?
+					<br/>
+					Well... We are making a streaming platform. An open-source streaming platform.
+					<br/>
+					You can join us on our journey of world conquer... It will be fun... Probably.
+				</p>
 			</div>
 		</div>
 	</div>
 </body>
+
+<style lang="scss">
```

Comment:
I think a lot of these styles can be moved to a globals.scss
Where we import it in the layout file for the entire application.

Rather than have them on the page.

Stuff like the body, h1 h2 ect 

Unless this is styles specific for this page?

Reply:
i'm mainly building these styles just for this page, can extract styles to a global level when other pages start getting built